app/views/articles/_video_player.html.erb

```diff
@@ -1,18 +1,26 @@
-<div itemscope itemtype="http://schema.org/VideoObject" class="video-player-header">
+<div class="video-player-header">
   <% if meta_tags %>
-    <meta itemprop="uploadDate" content="<%= article.published_at %>" />
-    <meta itemprop="name" content="<%= article.title %>" />
-    <meta itemprop="description" content="<%= article.description %>" />
-    <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
-    <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
     <% minutes, seconds = article.video_duration_in_minutes.split(":") %>
-    <meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
     <div id="video-player-source" data-source="<%= article.video_source_url %>"></div>
   <% end %>
   <script src="//content.jwplatform.com/libraries/b1zWy2iv.js" async></script>
   <div id="video-player-<%= article.id %>" class="video-player"></div>
 </div>
 
+<script type="application/ld+json">
```

Comment:
do you know if this should technically be in the document's `<head>` ?

Reply:
I do know! 😄 It is okay to place the JSON-LD in both the document's `<head>` and `<body>`, but the `<head>` is recommended.

![Screen Shot 2020-04-03 at 1 29 47 PM](https://user-images.githubusercontent.com/32834804/78397882-470d4500-75af-11ea-9600-3814b83787a2.png)